hera_sim/visibilities/conversions.py

```diff
@@ -64,16 +64,27 @@ def uvbeam_to_lm(uvbeam, freqs, n_pix_lm=63, trunc_at_horizon=False, **kwargs):
 
     az = -np.arctan2(m, l)
     za = np.arcsin(n)
-
+    
+    # This does not work at the moment
+    ####uvbeam.efield_to_power()
```

Comment:
This will work once https://github.com/RadioAstronomySoftwareGroup/pyuvsim/pull/232 is merged (hopefully today)

Reply:
Awesome! Yes I tried and it does work now.